### ISSUE TITLE
[Feature] Share Link per calendar, not per user

### ISSUE THREAD

**Issue by Dubidubiduu** [SergeyMosin/Appointments#81]
Setting:
So, small NGO again, we use Appointments for our onbording process. Right now, I create the appointments for the onboarding-calendar, of which the board of the ngo has access to. Then there is the auto-mail, sending a link to the new persons to make an appointment.

Problem:
No one else can create new appointments, since the link is based on the account who created the appointments, not on the calendar*. Or the created appointments of my colleagues wouldn't fit the link in the mail.

Alternate:
Create a new account just to create appointments and give credentials to everyone of the board. But this is gonna be a mess with my not tekki colleagues.

Preferred Solution:
So, I would prefer calendar-based links over account based links. This would also give me the opportunity to create multiple appoinment situations (onboarding, internal, external)....

* if I don't get everything wrong

**Comment by SergeyMosin** [comment]
Have you tried sharing the calendar in the Calendar app. Shared calendars are supported since a few versions back.

![shared_calendar](https://user-images.githubusercontent.com/10747559/82559863-5552f880-9b3e-11ea-9745-d123003036c8.jpg)

I would create a calendar and then share (with write access) the calendar with your colleagues. They can select it in the appointments app and add appointments. Only ONE of you should enable the "Public Appointments Page", just use the same public URL.

**Comment by hferee** [comment]
Note : #21 would also be a (partial) solution for me, but less satisfactory (and doesn't seem easier to implement).

**Comment by SergeyMosin** [comment]
This is a major feature so current target is probably v1.5* or even v1.6.* but it is definitely doable.

**Comment by nolens** [comment]
> So I was expecting the app to work this way: for each calendar, being able to publish one registration page.

I have the same need and would support the idea. In my case, I have different tasks to do with clients that need different times of day and different length of slots. So I want to share different "slot types".

So having multiple calendars and multiple appointment pages for each "slot type" would be awesome.

**Comment by LittleDuke** [comment]
To re-iterate the ask:

I'd like to have a calendar that offered various "slots" but would then block out others.

For example:  We offer free 20-minute consultations but also would like to be able to offer 55-minute meetings to existing clients.  If someone books a 20-minute slot in one calendar or in the same calendar but with overlapping slots we would like the system to remove the availability to schedule the 55-minute appointment.

You see something very similar to this with massage therapy calendars where someone may want a 30 minute appointment but the owner would also like to offer full hour and two hour slots.

Basically two different landing pages with two different views on the calendar(s)

**Comment by nolens** [comment]
This is now possible with version 1.7

Please close this issue.
